main.go

```diff
@@ -121,6 +121,19 @@ func main() {
 		ws.smgr = base.smgr
 		ws.store = base.storage
 
+		if v := os.Getenv("SECURE_KEY"); v != "" && secureKeyFlag == "" {
+			secureKeyFlag = v
+		}
+		if v := os.Getenv("OT_PUBLISH_PASSWORD"); v != "" && otPassword == "" {
+			otPassword = v
+		}
+		if v := os.Getenv("AUTH_CLIENT_SECRET"); v != "" && ah.ClientSecret == "" {
+			ah.ClientSecret = v
+		}
+		if v := os.Getenv("FSQ_CLIENT_SECRET"); v != "" && ah.ClientSecret == "" {
```

Comment:
```suggestion
		if v := os.Getenv("FSQ_CLIENT_SECRET"); v != "" && ws.fsqOauthConfig.ClientSecret == "" {
```

Is this a copy & paste error?